modules/git/src/index.ts

```diff
@@ -175,7 +184,11 @@ export type ModifiedStaged = {
  * const betweenRefs = await git.getModifiedFiles('v1.2.3', 'v2.0.0');
  * ```
  */
-export async function getModifiedFiles(modified: ModifiedStaged | ModifiedFromThrough = {}, options: Options = {}) {
+export async function getModifiedFiles(
+	modified: ModifiedStaged | ModifiedFromThrough = {},
+	options: Options = {},
+	includeDeletions = false,
```

Comment:
To follow general convention, let's merge `includeDeletions` into the `options: Options`. This keeps all of the methods from ballooning with too many unnamed arguments.

```ts
export type ModifiedFilesOptions = Options & { 
  /**
   * By default, this method will exclude files that were deleted to avoid accidentally attempting to operate on something that doesn't exist. To include the files in the output list, set this option to `true`.
   * @default `false`
   */
  includeDeletions?: boolean;
};

export async function getModifiedFiles(
  modified: Modified | ModifiedFromThrough = {},
  options: ModifiedFilesOptions = {};
) {
	const { from, staged, through } = modified;
	const { includeDeletions = false, step } = options;
// ...
```